content/60.methods.md

```diff
@@ -16,37 +16,50 @@ In an effort to keep pace as new information about COVID-19 and SARS-CoV-2 becom
+Similarly, many literature reviews have been written on the available COVID-19 literature [@doi:10.1016/j.molmed.2020.02.008; @doi:10.1016/j.immuni.2020.05.002; @doi:10.1126/scitranslmed.abc1931; @doi:10.1016/j.immuni.2020.05.002; @doi:10.1001/jama.2020.6019; @doi:10.1038/d41591-020-00026-w; @doi:10.1001/jama.2020.12839].
+However, static reviews quickly become outdated as new research is released or existing research is retracted or superseded
 Therefore, the COVID-19 publishing climate presented a challenge where curation of the literature by a diverse group of experts in a format that could respond quickly to high-volume, high-velocity information was desirable.
 
+We therefore sought to develop a platform for scientific discussion and collaboration around COVID-19 by adapting open publishing infrastructure to accommodate the scale of the COVID-19 publishing boom.
 Recent advances in open publishing have created an infrastructure that facilitates distributed, version-controlled collaboration on manuscripts [@doi:10.1371/journal.pcbi.1007128].<!--To Do: possibly cite some other efforts here-->
 Manubot [@doi:10.1371/journal.pcbi.1007128] is a collaborative framework developed to adapt open-source software development techniques and version control for manuscript writing.
 With Manubot, manuscripts are managed and maintained using GitHub, a popular, online version control interface that also provides the infrastructure via continuous integration (CI) to incorporate code into the manuscript building process to allow, for example, figures to be continuously updated based on an external data set.
-This open-publishing platform has been used to develop large-scale collaborative efforts such as <!--To Do: brief summary of deep review and one other use case-->.
-However, although synthesis and discussion of the emerging literature by biomedical scientists and clinicians would be expected to provide novel insights into how various areas of COVID-19 research intersect, such tools are not typically associated with biomedical research and the reliance on git can present a significant technical barrier to entry for biomedical scientists.
-The problem of synthesizing the COVID-19 literature lends itself well to a crowd-sourced approach to writing through open collaboration, but in biology, such efforts often rely on WYSIWYG tools such as Google Docs despite the significant limitations of such approaches.<!--To Do: recheck what the Manubot paper says about this and possibly pull some other refs-->
-Therefore, in addition to the unprecedented release of information, COVID-19 presents a unique challenge because most subject matter experts have limited technical training.
+This open-publishing platform has been used to develop large-scale collaborative efforts such as a review of developments in deep learning [@doi:10.1098/rsif.2017.0387].
```

Comment:
This is not essential but https://doi.org/10.1080/08989621.2020.1779591 is another example large scale collaboration.

Reply:
```suggestion
This open-publishing platform has been used to develop large-scale collaborative efforts such as a review of developments in deep learning [@doi:10.1098/rsif.2017.0387] and a re-evaluation of the role of authorship in modern collaborations [@doi:10.1080/08989621.2020.1779591].
```